packages/design-system-react/tsconfig.build.json

```diff
@@ -7,7 +7,7 @@
     "jsx": "react",
     "types": ["react"],
     "lib": ["ES2020", "DOM"],
-    "skipLibCheck": true
+    "skipLibCheck": true // TODO: Remove this once we have a proper type definition for the blo module for blockies
```

Comment:
Adding comment to show that this is for the `blo` module and can be removed if we move blockies into the monorepo